.github/scripts/auto_review.py

```
@@ -86,12 +121,45 @@ def post_pr_comment(repo, pr_number, body, github_token):
+다음 형식으로 작성해주세요:
+1. 변경 사항 요약
+2. 주요 변경 내용"""
+    
+    client = OpenAI(api_key=OPENAI_API_KEY)
+    response = client.chat.completions.create(
+        model="gpt-4o-mini",
+        messages=[{"role": "user", "content": prompt}]
+    )
+    return response.choices[0].message.content
+
+def update_pr_description(repo, pr_number, description, github_token):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)